acquia/cli
# DX-3196: More helpful error message if curl is missing

Motivation
----------
Fixes #393

Proposed changes
---------
Update Amplitude to accommodate https://github.com/zumba/amplitude-php/pull/28

Now if the curl extension is missing, you get a more informative error prior to ACLI running:
![Screenshot 2021-01-26 150055](https://user-images.githubusercontent.com/1984514/105917771-af6a5e80-5fe7-11eb-9aec-12663ee5483e.png)

Testing steps
---------
How can we replicate the issue and verify that this PR fixes it?

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `acli ckc`
3. Build the phar (you cannot test this from source since it relies on Box dependency checking)
4. Run the phar in Windows without php-curl enabled and verify the error output is as above.
